.changeset/silly-walls-open.md

```diff
@@ -0,0 +1,33 @@
+---
+'@directus/types': patch
+'@directus/env': patch
+'@directus/api': patch
+'create-directus-extension': patch
+'create-directus-project': patch
+'@directus/extensions': patch
+'@directus/extensions-sdk': patch
+'@directus/memory': patch
+'@directus/system-data': patch
+'@directus/themes': patch
+'@directus/update-check': patch
+'@directus/utils': patch
```

Comment:
I believe this is missing a few dependencies, for example `@directus/extensions` is also depended on by `@directus/extensions-registry`. `@directus/utils` and `@directus/types` definitely have more dependent packages as well.